src/Evaluators.h

```diff
@@ -8,21 +8,95 @@
 #include <vector>
 // TODO: Better evaluation
 #include <random>
+#include <iostream>
 
 static std::random_device rd;
 static std::uniform_real_distribution<double> u_dist;
 
 using DS = DomineeringState;
 
+// TO DO: make helper methods.
+
 struct EvalTakeAway {
     double operator()(const DS& state) const {
-        return u_dist(rd);
+	int count = 0;
```

Comment:
Make sure you use 4 spaces for indentation. If you do `gg=G` in Vim, you should be able to fix them at once.